app/v1/models/request/listForeignPropertes/ListForeignPropertiesRequest.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.request.listForeignPropertes
+
+import uk.gov.hmrc.domain.Nino
+
+case class ListForeignPropertiesRequest(nino: Nino, businessId: String, fromDate: String, toDate: String)
```

Comment:
I believe the dates should be optional

Reply:
Dates are optional in the raw data but we default the dates if they're not provided, so dates will always be sent down to DES.

Reply:
Gotcha